packages/card/index.tsx

```diff
@@ -101,6 +113,25 @@ function Heading({ as: Component = 'h2', children }: HeadingProps) {
   )
 }
 
+function ProductBadges({ badges, appearance = 'light' }: ProductBadgesProps) {
```

Comment:
Cards can only render productBadges with the dot and secondary theme?

Reply:
See my comment above in the PR description.

Tangentially related, but `hasDot` is the `secondary` theme plus a dot versus having totally separate styles if that makes sense.

Reply:
Got it, sorry I missed that in the description!

Thoughts on simplifying the prop if we're being restrictive here.

```tsx
<Card
  productBadges={['vault', 'consul']}
/>
```